index.css

```
@@ -710,3 +714,135 @@ body a > .commit-hash {
+
+.theme-card {
+    border: 2px solid transparent;
+    border-radius: 8px;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0.5rem;
+    gap: 0.4rem;
+    background: var(--fallback-background-color);
+    background: var(--background-color);
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)